_assignments/mp1.md

```diff
@@ -0,0 +1,395 @@
+The room should be 15x15x15 Unity units (aka meters).
+>
+> We will award extra credit for [more complicated geometry](#a-more-interesting-room).
+
+First, create a plane. It can be found in the top bar menus, under `GameObject` → `3D Object` → `Plane`, as shown below.
+
+![Create plane](/img/assignments/mp1/image3.png)
+
+Unity's default plane size is $$10 \times 10 \hspace{0.5em} (X \times Z)$$ units. In order to make your room 15 units wide, you have to scale the plane. On the right side (in the default editor layout) you will find the Inspector window. This window provides details about the currently selected object. Select the plane in the Scene view, and the Inspector will fill with information and settings for said plane. Find the “Scale” option, and set it to 1.5 on the $$ X $$ and $$ Z $$ to make your plane 15 units wide and long.
+
+![Inspector](/img/assignments/mp1/image7.png)
+
+Note: The plane has no thickness, so the value in $$ Y $$ can be any positive integer.
+
+Now, to easily make the walls and ceiling, you can select your plane in the hierarchy, and duplicate it.
+
+![Duplicate](/img/assignments/mp1/image4.png)
+
+From there, simply change the new plane’s rotation and position to make it one of the walls or ceilings.
+Unity measures position from the center of the object, so if you want your walls to match up with the floor (at height 0), your walls will need to be at $Y = 7.5$.
+
+![Scene window](/img/assignments/mp1/image15.png)
+
+Note: In the image above, rotating the plane also rotated its axes (the blue z axis now points down).
+Make sure to account for that when rotating and moving objects!
+
+Note: By default, your scene has a directional light in it, which illuminates your entire scene from a specified angle, from very far away, much like the sun.
+You’ll notice that your planes do not block this light.
+That’s because planes only block light (and render) from one side. Bear this in mind when creating objects in Unity in the future!
+For now, just delete the directional light. You will add more lights in later.
+
+## VR Player Controller
+
+> Place an OVR Player Controller prefab in the room.
+This prefab handles basic movement, collision, and camera control.
+
+The lab computers come with the Oculus OVR plugin installed.
+Go to `Assets` → `Import Package` → `OculusUtilites`, and then click "import".
+
+![Import OculusUtilities](/img/assignments/mp1/image20.png)
+
+If you can't find the package, go to the [Oculus Website](https://developer.oculus.com/downloads/package/oculus-utilities-for-unity-5/) and download their Unity package.
+Unzip the unity package file, and then go to `Assets` → `Import Package` → `Custom Package`.
+Find the unzipped `OculusUntilites.unitypackage`, and import all of the items within.
+You should now have an `OVR` folder in your `Assets` folder.
+
+In this class, you will be primarily using two prefab objects from this package, both found in `OVR` → `Prefabs`.

```

Comment:
update paths with `Oculus` (after confirming instructions for package installation)